src/DssSpell.sol

```diff
@@ -18,16 +18,52 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import "dss-interfaces/dss/GemJoinAbstract.sol";
+import "dss-interfaces/ERC/GemAbstract.sol";
 
-import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3MAavePoolConfig, D3MAaveBufferPlanConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MCoreInstance } from "src/dependencies/dss-direct-deposit/D3MCoreInstance.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
+interface Initializable {
+    function init(bytes32) external;
+}
+
+interface RwaLiquidationLike {
```

Comment:
Name fits conventions here but doesn't match how we do in dss-interfaces. Note here for me to change our convention to be `RwaLiquidationOracleLike` to match the onchain name of the interface.

cast interface 0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30

Will give `RwaLiquidationOracle` which should be `RwaLiquidationOracleLike` but this isn't demonstrated in the archives. We can debate this.

Reply:
Have no opinion here, can change it if needed